### PR TITLE
[HOTFIX] fix conveyorbelt icons

### PR DESCRIPTION
**Pull request by spookydonut**
another oopsie

### CODE REVIEW

**Review by ghost — APPROVED** [review]
And they don't stop coming and they don't stop coming and they don't stop coming

File: code/modules/recycling/conveyor2.dm [hunk 1]
```diff
@@ -116,6 +118,7 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
 /obj/machinery/conveyor/proc/update()
 	if(stat & BROKEN || !operable || stat & NOPOWER)
 		operating = FALSE
+		update_icon()
```
Comment (ShizCalev) [inline code/modules/recycling/conveyor2.dm]: this should probably be called either way tbh

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/modules/recycling/conveyor2.dm (hunk 1 above)
Comment (spookydonut) [reply code/modules/recycling/conveyor2.dm]: It's already called where the value changes and doing it here stops it being called twice

Comment (cacogen) [reply code/modules/recycling/conveyor2.dm]: to think you want to call this twice but haven't called your mother once

**Pull request merged**